js/interface/map-panel.js

```diff
@@ -78,4 +81,22 @@ Fliplet.InteractiveMap.component('map-panel', {
   destroyed() {
     Fliplet.InteractiveMap.off('maps-save', this.onInputData)
   }
-});
\ No newline at end of file
+});
+
+Fliplet.Widget.onCancelRequest(function () {
+  var providersNames = [
+    'filePickerProvider',
+    'iconPickerProvider'
+  ];
```

Comment:
No semicolons please

Reply:
@tonytlwu Removed semicolons. Sorry for the inconvenience.